src/rest/zone_client.erl

```diff
@@ -60,118 +61,179 @@ index_ready(Zone) when is_binary(Zone) ->
+    Url = http_utils:build_url(Address, <<"graph/edges/permissions">>, Params),
+    http_executor:post_request(Url, false).
+
+% MUST
+-spec add_vertex(Zone:: binary(), Name:: binary(), Type:: binary()) -> ok | {error, any()}.
+add_vertex(Zone, Name, Type) ->
+    {ok, Address} = http_utils:get_address(Zone),
+    Url = http_utils:build_url(Address, <<"graph/vertices">>,
+      [{<<"type">>, Type}, {<<"name">>, Name}]),
+    http_executor:post_request(Url, false).
 
 -spec add_vertices(Zone:: binary(), BulkRequest:: map()) -> ok | {error, any()}.
-add_vertices(_Zone, _BulkRequest) ->
+add_vertices(Zone, BulkRequest) ->
     {error, not_implemented}.
 
 -spec post_event(VertexId:: binary(), Event:: map()) -> ok | {error, any()}.
-post_event(_VertexId, _Event) ->
+post_event(VertexId, Event) ->
     {error, not_implemented}.
 
 -spec post_events(Zone:: binary(), BulkMessages:: map()) -> ok | {error, any()}.
-post_events(_Zone, _BulkMessages) ->
+post_events(Zone, BulkMessages) ->
     {error, not_implemented}.
 
 -spec get_event_stats(Zone:: binary()) -> {ok, map()} | {error, any()}.
-get_event_stats(_Zone) ->
+get_event_stats(Zone) ->
     {error, not_implemented}.
 
 -spec get_dependent_zones(Zone:: binary()) -> {ok, map()} | {error, any()}.
 get_dependent_zones(Zone) ->
     get_dependent_zones(Zone, []).
 
 -spec get_dependent_zones(Zone:: binary(), ToExclude:: list(binary())) -> {ok, map()} | {error, any()}.
-get_dependent_zones(_Zone, _ToExclude) ->
+get_dependent_zones(Zone, ToExclude) ->
     {error, not_implemented}.
 
 -spec is_instrumentation_enabled(Zone:: binary()) -> {ok, boolean()} | {error, any()}.
-is_instrumentation_enabled(_Zone) ->
+is_instrumentation_enabled(Zone) ->
     {error, not_implemented}.
 
 -spec set_instrumentation_enabled(Zone:: binary(), Enabled:: boolean()) -> ok | {error, any()}.
-set_instrumentation_enabled(_Zone, _Enabled) ->
+set_instrumentation_enabled(Zone, Enabled) ->
     {error, not_implemented}.
 
 -spec set_indexation_enabled(Zone:: binary(), Enabled:: boolean()) -> ok | {error, any()}.
-set_indexation_enabled(_Zone, _Enabled) ->
+set_indexation_enabled(Zone, Enabled) ->
     {error, not_implemented}.
 
 -spec simulate_load(Zone:: binary(), LoadRequest:: map()) -> ok | {error, any()}.
-simulate_load(_Zone, _LoadRequest) ->
+simulate_load(Zone, LoadRequest) ->
     {error, not_implemented}.
 
 %% function that blocks process until zone (all zones) returns 'true' from 'IP/index_ready' endpoint
 %%  - delay between consecutive requests may be constant, or Exponential Backoff can be used, whatever
 -spec wait_for_index(Zones:: binary() | list(binary()), Timeout:: integer()) -> ok | {error, any()}.
-wait_for_index(Zone, _Timeout) when is_binary(Zone) ->
+wait_for_index(Zone, Timeout) when is_binary(Zone) ->
     {error, not_implemented};
 
-wait_for_index(Zones, _Timeout) when is_list(Zones) ->
+wait_for_index(Zones, Timeout) when is_list(Zones) ->
```

Comment:
Ostatnie z serii tych kilku(nastu) funkcji: czemu to się kompiluje pomimo nieużywanego parametru? Jeśli to nie będzie robiło problemów, to spoko, ale jestem zaskoczony